client/packages/common/src/ui/layout/tables/material-react-table/useGetColumnDefDefaults.tsx

```diff
@@ -73,14 +75,19 @@ export const useGetColumnTypeDefaults = () => {
 
       case ColumnType.Comment:
         return {
-          size: 60,
+          size: 50,
           enableSorting: false,
           enableColumnFilter: false,
           enableResizing: false,
           enableColumnActions: false,
-          // Displays an empty header in the table, but still uses the provided
-          // "header" prop for the Column Management menu
-          Header: () => <></>,
+          // Comment popover is pretty narrow, show icon rather than full label
+          Header: () => (
+            <Tooltip title={t('label.comment')} placement="top">
+              <div>
+                <MessageSquareIcon fontSize="small" />
```

Comment:
found the blank space confusing, especially where comment is not the final column, and no comments may be set, so it's just a big blank space - but text label is a bit wide... how's this?

<img width="248" height="123" alt="Screenshot 2025-10-08 at 11 00 27 AM" src="https://github.com/user-attachments/assets/eba9f4e4-9d79-448b-89bb-02bdf087a21b" />